modules/fee-market/rpc/runtime-api/src/lib.rs

```diff
@@ -73,9 +73,7 @@ decl_runtime_apis! {
 	where
 		Balance: Codec + MaybeDisplay + MaybeFromStr,
 	 {
-		fn market_fee(
-		) -> Option<Fee<Balance>>;
-
-		fn in_process_orders() -> InProcessOrders;
+		fn market_fee(instance: u8) -> Option<Fee<Balance>>;
```

Comment:
market fee api will be removed. keep it here?

Reply:
Keep it here right now. I'll clean it after the storage-read functionality is stable on the SDK side.

Reply:
After confirm with apps and bridger, these rpc can be safely removed.